packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md

```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
 
 You can test your integration by configuring a [routing rule](https://help.okta.com/okta_help.htm?id=ext-cfg-routing-rules) to use <StackSelector snippet="idp" noSelector inline /> as the Identity Provider.
 
-### Use the Authorize URL to simulate the authorization flow
-
-You can also use the Authorize URL to simulate the authorization flow. The Okta Identity Provider that you created generated an authorize URL with a number of blank parameters that you can fill in to test the flow with the Identity Provider. The authorize URL initiates the authorization flow that authenticates the user with the Identity Provider.
+Alternatively, you can use the Authorize URL to simulate the authorization flow. The Okta Identity Provider that you created generated an authorize URL with a number of blank parameters that you can fill in to test the flow with the Identity Provider. The authorize URL initiates the authorization flow that authenticates the user with the Identity Provider.
 
-> **Note:** Use this step to test your authorization URL as an HTML link. For information on using the Sign-In Widget, Okta-hosted sign-in page, or AuthJS, see [Use the Identity Provider to sign in](/docs/guides/add-an-external-idp/apple/main/#use-the-identity-provider-to-sign-in).
+> **Note:** Use this step to test your authorization URL as an HTML link. For information on testing your authorization URL using the Sign-In Widget, Okta-hosted sign-in page, or AuthJS, see the [next section](/docs/guides/add-an-external-idp/-/main/#use-the-identity-provider-to-sign-in).
```

Comment:
```suggestion
> **Note:** Use this step to test your authorization URL as an HTML link. For information on testing your authorization URL using the Sign-In Widget, Okta-hosted sign-in page, or AuthJS, see the [next section](#use-the-identity-provider-to-sign-in).
```
I'm not sure if you need the full path? If the full path is required for the structure of these docs, then just ignore this suggestion.